.cirrus.yml

```diff
@@ -109,13 +109,24 @@ task:
         matrix:
           CHANNEL: "master"
           CHANNEL: "stable"
-      tool_script:
+      analyze_tool_script:
         - cd script/tool
         - dart analyze --fatal-infos
-      script:
+      analyze_script:
         # DO NOT change the custom-analysis argument here without changing the Dart repo.
         # See the comment in script/configs/custom_analysis.yaml for details.
         - ./script/tool_runner.sh analyze --custom-analysis=script/configs/custom_analysis.yaml
+      pathified_analyze_script:
+        # Re-run analysis with path-based dependencies to ensure that publishing
+        # the changes won't break analysis of other packages in the respository
+        # that depend on it.
+        - ./script/tool_runner.sh pathify --target-non-breaking-update-packages
```

Comment:
Related to my other comment, unfortunately I think this is harder to grok than it could be.

`./script/tool_runner.sh convert_dependencies_to_paths --convert-non-breaking-updated-packages` is slightly better.